ui/src/views/Documentation/index.jsx

```diff
@@ -75,10 +75,9 @@ export default class Documentation extends Component {
   }
 
   findChildFromRootNode(node) {
-    const currentPath = window.location.pathname.replace(
-      `${DOCS_PATH_PREFIX}/`,
-      ''
-    );
+    const currentPath = window.location.pathname
+      .replace(/\/+$/, '')
```

Comment:
What's the reason for adding `+`? I wonder if `.replace(/\/$/, '')` is all we need.

Reply:
@helfi92 You are right. No need for the +. I removed it.